api/service_instance_controller.go

```diff
@@ -42,7 +42,7 @@ func NewServiceInstanceController(ctx context.Context, options *Options) *Servic
 	return &ServiceInstanceController{
 		BaseController: NewAsyncController(ctx, options, web.ServiceInstancesURL, types.ServiceInstanceType, true, func() types.Object {
 			return &types.ServiceInstance{}
-		}),
+		}, true),
```

Comment:
do we block creation of new bindings while instance is being cascade-deleted, like we do for platforms? I didn't see any code doing that or test covering it

Reply:
1) Cascade delete functionality automatically creates sub operations for broker/instances deletion when platform deletion is requested.
2) Once delete operation exists for broker/instance, it is already protected by delete handling logic. For example: https://github.com/Peripli/service-manager/blob/700d4b92ac9159ce165c6faea87a1fd5009f75c8/storage/interceptors/smaap_service_binding_interceptor.go#L140
3) We have a set of requests (including binding, unbinding and etc...) with check for expected response code. We are expecting to get unprocessable entity when calling for instance binding during platform deletion - https://github.com/Peripli/service-manager/blob/50d53f181f744b5ed6c3bfc08c9616d9d7f4a77a/test/osb_and_plugin_test/plugin_test/platform_termination_plugin_test.go#L101.